octobot_trading/data/order.py

```diff
@@ -248,12 +249,18 @@ def get_currency_and_market(self) -> (str, str):
     def get_total_fees(self, currency):
         return get_fees_for_currency(self.fee, currency)
 
+    def is_open(self):
+        return self.status in [OrderStatus.OPEN, OrderStatus.PARTIALLY_FILLED]
+
     def is_filled(self):
         return self.status is OrderStatus.FILLED
 
     def is_cancelled(self):
         return self.status is OrderStatus.CANCELED
 
+    def is_closed(self):
+        return self.status in [OrderStatus.CANCELED, OrderStatus.FILLED, OrderStatus.CLOSED]
```

Comment:
I think it should be a set instead of a list

Reply:
Why ?